### ISSUE TITLE
Handle errors in squares-pay import better

### ISSUE THREAD

**Issue by dehnert** [tech-squares/squaresdb#23]
In general, if the import from squares-pay is bad, we should display an error message, not just 500.

**Comment by dehnert** [comment]
One notable class of error we should handle is people choosing different options for the export -- not form keys for headers, wrong select list format, long options, etc. -- I think current dev work may focus on correct export but dirty data from the payer.

**Comment by dehnert** [comment]
Adding a link to the export instructions on https://wikis.mit.edu/confluence/display/techsquares/Squares+Pay would also be clever. For that matter, for "staff", the navbar should probably just link to https://wikis.mit.edu/confluence/display/techsquares/SquaresDB

**Comment by dehnert** [comment]
A bunch of progress:
```
commit d944528407075924c3893cb7bb6afe33ec787212
Author: Alex Dehnert <adehnert@mit.edu>
Date:   Sat Jun 17 20:52:21 2023 -0400

    Sub upload: Better error handling
    
    - Error and warning mechanism (#23)
    - Warn on rejected payments (and don't give them a sub) (#22)
    - Error message on unexpected sub period (maybe this should be a
      warning?)
    - Error message on unknown person
    - Add a couple type annotations
```

**Comment by dehnert** [comment]
> One notable class of error we should handle is people choosing different options for the export -- not form keys for headers, wrong select list format, long options, etc. -- I think current dev work may focus on correct export but dirty data from the payer.

I think these are fixed in commit f9f1e2c59 (and 6b8eda).

At this point, remaining issues can be treated as a new issue.
